chart/questions.yaml

```
@@ -82,6 +82,18 @@ questions:
     type: string
     label: Longhorn Backing Image Manager Image Tag
     group: "Longhorn Images Settings"
+  - variable: image.longhorn.supportBundleKit.repository
+    default: longhornio/support-bundle-kit
+    description: "Specify Longhorn Support Bundle Kit Image Repository"
```

Sorry for the late review.

nit: a little inconsistent with the doc: https://longhorn.io/docs/1.4.0/references/settings/#support-bundle-manager-image
```suggestion
    description: "Specify Longhorn Support Bundle Manager Image Repository"
```

---

understood. I created another PR, https://github.com/longhorn/longhorn/pull/5276, to update the description. Thank you.